app/code/Magento/Search/Test/Unit/Model/PopularSearchTermsTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Search\Test\Unit\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Search\Model\PopularSearchTerms;
+use Magento\Search\Model\ResourceModel\Query\Collection;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * @covers \Magento\Search\Model\PopularSearchTerms
+ */
+class PopularSearchTermsTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Testable Object
+     *
+     * @var PopularSearchTerms
+     */
+    private $popularSearchTerms;
+
+    /**
+     * @var ScopeConfigInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $scopeConfigMock;
+
+    /**
+     * @var Collection|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $queryCollectionMock;
+
+    /**
+     * Set Up
+     *
+     * @return void
+     */
+    protected function setUp()
+    {
+        $this->scopeConfigMock = $this->createMock(ScopeConfigInterface::class);
```

Comment:
Just a recommendation for the future. If you are not going to reuse properties in different methods, it's a good idea to use local method variables instead of class properties.